### DISCUSSION TITLE
Mixing of HTTP and HTTPS URLs error

### DISCUSSION THREAD

**Discussion by kamranayub** [ravendb/ravendb#15882] [Q&A] [answered]
This may be specific to the Node.js client but I am testing connecting to `live-test.ravendb.net` without a certificate. If I use that HTTP URL, I am seeing this error when trying to initialize the DocumentStore:

```
The url http://live-test.ravendb.net is using HTTP, but other urls are using HTTPS, and mixing of HTTP and HTTPS is not allowed.
```

This is with `ravendb@5.2.7`

cc @ml054

**Comment by ayende** [comment, marked as answer]
Are you still passing the `Certificate` value?
That is not allowed, and the error is likely confusing in this case.

**Reply by kamranayub** [reply]
Ah, that was it -- I was passing `{}` to `authOptions` if there was no certificate provided but if I switch it to pass `undefined` it works! Thank you.
